jenkins/automation/kubernetes/k8-cicd-motr.groovy

```diff
@@ -0,0 +1,334 @@
+pipeline {
+	agent {
+		node {
+			label "docker-${OS_VERSION}-node"
+		}
+	}
+	options {
+		timeout(time: 180, unit: 'MINUTES')
+		timestamps()
+		skipDefaultCheckout()
+	}
+	parameters {
+		string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Repo for Motr')
+		string(name: 'MOTR_BRANCH', defaultValue: 'kubernetes', description: 'Branch for Motr')
+		choice (
+			choices: ['DEBUG', 'ALL', 'DEVOPS'],
+			description: 'Email Notification Recipients ',
+			name: 'EMAIL_RECIPIENTS'
+		)
+	}
+	environment {
+		CORTX_RE_BRANCH = "kubernetes"
+		CORTX_RE_REPO = "https://github.com/Seagate/cortx-re/"
+
+		GPR_REPO = "https://github.com/${ghprbGhRepository}"
+		MOTR_URL = "${ghprbGhRepository != null ? GPR_REPO : MOTR_URL}"
+		MOTR_BRANCH = "${sha1 != null ? sha1 : MOTR_BRANCH}"
+		MOTR_GPR_REFSEPEC = "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*"
+		MOTR_BRANCH_REFSEPEC = "+refs/heads/*:refs/remotes/origin/*"
+		MOTR_PR_REFSEPEC = "${ghprbPullId != null ? MOTR_GPR_REFSEPEC : MOTR_BRANCH_REFSEPEC}"
+		
+		//////////////////////////////// BUILD VARS //////////////////////////////////////////////////
+		// OS_VERSION, hosts and COMPONENTS_BRANCH are manually created parameters in jenkins job.
+		
+		COMPONENT_NAME = "motr".trim()
+		BRANCH = "${ghprbTargetBranch != null ? ghprbTargetBranch : COMPONENTS_BRANCH}"
```

Comment:
Is this same as `MOTR_BRANCH`

Reply:
Removed COMPONENTS_BRANCH parameter using motr branch parameter as COMPONENTS_BRANCH parameter in script.